pom.xml

```diff
@@ -9,6 +9,7 @@
 	<version>1.0.0-SNAPSHOT</version>
     <modules>
         <module>lombok-custom</module>
+        <module>reactive-modules</module>
```

Comment:
Looks like lombok custom is in the wrong place. :) You'll see a much longer list of modules later on in the pom. Please place `reactive-modules` there.

Reply:
Is profile, 'default-first' the right place to place the module?